File: 07- Idade_Dias/ex07.cs
/// Faça um código em que leia a idade de uma pessoa expressa em anos, meses e dias e escreva a idade dessa pessoa expressa apenas em dias.
/// 

using System;

class Program {

public static void Main (string[] args) {

int a, m, d, A, M, D;

Console.WriteLine ("Informe a sua idade (anos): ");

a = int.Parse(Console.ReadLine());

Console.WriteLine ("Informe a sua idade (meses): ");

m = int.Parse(Console.ReadLine());

Console.WriteLine ("Informe a sua idade (dias): ");

d = int.Parse(Console.ReadLine());

A = 365 * a;

M = 30 * m;

D = A + M + d;

Console.WriteLine ("Você tem "+D+" dias");

}

}

File: 07- Idade_Dias/ex07.py
''' Faça um código que leia a idade de uma pessoa expressa em anos, meses e dias e escreva a idade dessa pessoa expressa apenas em dias.'''

a = (int)(input("Digite sua idade em anos: "))
m = (int)(input("Digite sua idade em meses: "))
d = (int)(input("Digite sua idade em dias: "))

anos = 365 * a
meses = 30 * m
dias = anos + meses + d

print("Sua idade em dias é: ",dias)
